build/spas.ts

```diff
@@ -13,7 +13,6 @@ import {
   BUILD_OUT_ROOT,
 } from "../libs/env";
 import { DocFrontmatter } from "../libs/types/document";
-// eslint-disable-next-line n/no-missing-require
```

Comment:
this disable doesn't apply because of the migration to import

Reply:
LGTM